src/main/java/uk/gov/companieshouse/insolvency/delta/mapper/CaseMapper.java

```diff
@@ -83,4 +84,18 @@ public void mapDatesAndLinks(@MappingTarget ModelCase modelCase,
             }
         });
     }
+
+
+    /**
+     * Invoked at the end of the auto-generated mapping methods and
+     * if the practitioners is null, then initialize it with ArrayList of Practitioners.
+     *
+     * @param modelCase     the target ModelCase object
+     */
+    @AfterMapping
+    public void initializePractitionersIfNull(@MappingTarget ModelCase modelCase) {
+        if (modelCase.getPractitioners() == null) {
+            modelCase.setPractitioners(new ArrayList<Practitioners>());
+        }
+    }
```

Comment:
Instead of this I'd try adding the following to the mapping annotation - I can try and let you know if it works as well.

In the mapping itself:
```@Mapping(target = "practitioners", source = "appointments")
```
 we could add a `nullValuePropertyMappingStrategy = NullValuePropertyMappingStrategy.SET_TO_DEFAULT`  after the source and that should generate an empty collection if the source value is null

Reply:
I saw this yesterday online but I was not sure we could apply this only for a specific attribute or this will be applied to all the attributes in the class

Reply:
Just tried it out, it only works on update methods not create - so those that have a `@MappingTarget` set, let me try another way and will get back to you

Reply:
It is possible but more verbose than your solution, so we can go with AfterMapping :)

Reply:
Thanks Chris